packages/app/src/cli/utilities/developer-platform-client/partners-client.ts

```diff
@@ -222,17 +223,25 @@ export class PartnersClient implements DeveloperPlatformClient {
   }
 
   async appFromId({apiKey}: MinimalAppIdentifiers): Promise<OrganizationApp | undefined> {
-    return fetchAppDetailsFromApiKey(apiKey, await this.token())
+    const app = await fetchAppDetailsFromApiKey(apiKey, await this.token())
+    if (app) app.developerPlatformClient = this
+    return app
   }
 
   async organizations(): Promise<Organization[]> {
     const result: AllOrganizationsQuerySchema = await this.request(AllOrganizationsQuery)
-    return result.organizations.nodes
+    return result.organizations.nodes.map((org) => ({
+      id: org.id,
+      businessName: org.businessName,
+      source: OrganizationSource.Partners,
+    }))
```

Comment:
This creates a 404 when the user doesn't have a Partners account. (Not sure if it can happen in production, but it definitely happens in spin.)

I think the simplest solution is to do this:

```suggestion
    try {
      const result: AllOrganizationsQuerySchema = await this.request(AllOrganizationsQuery)
      return result.organizations.nodes.map((org) => ({
        id: org.id,
        businessName: org.businessName,
        source: OrganizationSource.Partners,
      }))
    } catch (error: unknown) {
      if ((error as {statusCode?: number}).statusCode === 404) {
        return []
      } else {
        throw error
      }
    }
```